### ISSUE TITLE
Suggestions and or questions

### ISSUE THREAD

**Issue by GeekPR1977** [ahodges9/LedFx#102]
Can ya'll make a Android and or iOs app for LedFx to run on these devices, would be SOO DAMN COOL!, not having to have a PC or Rpi, some kick-ass app! maybe even install in a car?!

Can this be installed on Volumio, or other , music players? why don't ya'll develop a plugin, for othe rmusic player software?! I would even pay$$, not a lot.. but a fair price

**Comment by ashishverma2614** [comment]
yeah only reason i dont use Wled and Ledfx together because of  need of PC if we can really have LedFx mobile app it will be much better and easy to use
